builtInFunctions/esdtMetaDataRecreate.go

```diff
@@ -161,6 +162,10 @@ func getEsdtInfo(
 		return nil, ErrInvalidEsdtValue
 	}
 
+	if esdtData.TokenMetaData == nil {
+		esdtData.TokenMetaData = &esdt.MetaData{}
```

Comment:
is this backward compatible? I do not think so if we had a previous store and load and re-store in the history of the main chain

Reply:
Yes, because this func is used only by the newly added built in functions.

Reply:
ok